gateway/src/node/mixnet_handling/receiver/listener.rs

```diff
@@ -42,6 +52,12 @@ impl Listener {
                 connection = tcp_listener.accept() => {
                     match connection {
                         Ok((socket, remote_addr)) => {
+                             if !self.allowed_ingress.is_allowed(remote_addr.ip()) {
+                                // TODO: perhaps this should get lowered in severity?
+                                warn!("received an incoming connection from {remote_addr}, but this address does not belong to any node on the previous layer - dropping the connection");
```

Comment:
yeah I think let's use INFO, since nothing is going wrong from the operators perspective. I'd imagine most operators run their nodes with RUST_LOG=warn and are only interested in stuff they need to be aware of